src/impl/locale.js

```diff
@@ -374,14 +394,14 @@ export default class Locale {
     this.eraCache = {};
 
     this.specifiedLocale = specifiedLocale;
-    this.fastNumbersCached = null;
+    this.fastNumbersCached = supportsFastNumbers(this);
+    this.isEnglishCached =
```

Comment:
Similarly, I agree we should cache this, but why not cache it inside `isEnglish`. That way if `isEnglish` is never called, we don't pay the performance penalty